Giovanni Manor Revamp.

Raptorian1 opened the pull request:
## About The Pull Request
Redesigns the lovely manor to be larger and fit the poor giovanni family who is cramped tighter than sardines. Adds more bedrooms, redesigns more rooms, adds new rooms such as the ballroom, secret torture room, wine cellar, butcher in cellar, secret giovanni family making room and adds offices that differ from the dons!

## Why It's Good For The Game

expands the map and adds plenty of secrets like in bloodlines.

## Testing Photographs and Procedure
<img width="671" height="478" alt="image" src="https://github.com/user-attachments/assets/0b588483-9a0d-4d7e-99c2-f15dea43ec81" />
<img width="819" height="666" alt="image" src="https://github.com/user-attachments/assets/f6d9ff49-e0fe-47f4-aae6-438a661a661c" />
<img width="855" height="668" alt="image" src="https://github.com/user-attachments/assets/b04f2b95-9eb3-481e-8b63-c9f1e326e5b3" />
<img width="865" height="657" alt="image" src="https://github.com/user-attachments/assets/1ad19a99-bd6d-49fa-a852-e00bfd0a3c30" />

## Changelog

:cl:
del: Removed things
qol: quality of life changes
map: mapping changes
/:cl:

AlmostLikeClockwork commented:
think you misclicked here

<img width="186" height="166" alt="image" src="https://github.com/user-attachments/assets/c448c787-7e16-46bf-9a1f-e1465eb67e60" />

Raptorian1 commented:
<img width="447" height="516" alt="image" src="https://github.com/user-attachments/assets/4c5cd806-72ad-4198-83d2-fd48034f0210" />

Marshmellow105 commented:
This looks great, but not entirely ready. I dont have it in me to go through it comprehensively, but some items of note that I suggest you go over the whole project looking for:

Missing decals, like the concrete corners around the fountain or the carpet trim on the door side, or the corners by the fountain
<img width="750" height="774" alt="image" src="https://github.com/user-attachments/assets/0534fdee-da5c-4b44-8aec-61d5ca0fabb8" />
<img width="798" height="774" alt="image" src="https://github.com/user-attachments/assets/3a845b70-0ea5-4e32-a208-c60b6235fd96" />

Light attachments, dont stick lights on halfwalls/hedges
<img width="260" height="601" alt="image" src="https://github.com/user-attachments/assets/4130b76e-a078-4605-8630-666cbdfeb7b6" />

decal continuity, I'd put a corner in or continue the decal into the wall like shown
<img width="211" height="286" alt="image" src="https://github.com/user-attachments/assets/183d15ce-2100-492e-ab31-9b24927f9253" />

misaligned curtains, I can tell you know what they should look like from the rest and this is just a mistake
<img width="286" height="128" alt="image" src="https://github.com/user-attachments/assets/7b064be0-120c-4d90-b7a5-d7630630da13" />

surgery room has to go, there is a reason lots of private medbay/surgery areas got gutted in the remap and that is to encourage people to call doctors or visit the hospital. [VERY] specific exceptions include garou (because I wouldnt hear the end of it) and endron (who have it in a laboratory)
<img width="581" height="401" alt="image" src="https://github.com/user-attachments/assets/78b43664-0d3a-4389-bc03-51e01389d604" />

same curtain stuff
<img width="940" height="562" alt="image" src="https://github.com/user-attachments/assets/80f026d9-4b83-4c3f-8e4d-f9e802eefaff" />

same decal stuff
<img width="364" height="297" alt="image" src="https://github.com/user-attachments/assets/ac293d1b-9f8d-4a0d-9838-28c9db903e02" />

I dont really like the bright parquetry floor in the party room with the rest of the manor being that darker wood, but I understand you have limited options.

Overall, insanely beautiful job, its clear to me you took a lot of hints from the existing design and tried to follow it which I really appreciate. Dont hesitate to ping me @ bomby on discord for any questions or another review.

Marshmellow105 reviewed (changes requested):
formality, lmk when you got to the above points

Raptorian1 commented:
will do, taking a short break

Raptorian1 commented:
> formality, lmk when you got to the above points

fixed all issues and issues not listed. Replaced the surgery room to just be a vault with empty duffel bags.
<img width="663" height="734" alt="image" src="https://github.com/user-attachments/assets/7cf36175-b5c5-4301-a43e-e90a508e226a" />

Raptorian1 commented:
all feedback was reviewed and solved.

Marshmellow105 commented:
> The purple tables were used in old map desks for REALLY rich people, it's not so bad, i think they look good. Old Ventrue and Tzim offices had these.

just dont loik em
<img width="300" height="300" alt="simple as" src="https://github.com/user-attachments/assets/54e07f69-1018-4d37-8524-66ba765d85e1" />

Raptorian1 commented:
done, ready for TM
